internal/ui/backup/text.go

```diff
@@ -144,3 +148,13 @@ func (b *TextProgress) Finish(snapshotID restic.ID, start time.Time, summary *Su
 		ui.FormatDuration(time.Since(start)),
 	)
 }
+
+// Quote filenames with funny characters in them.
+func quote(filename string) string {
+	for _, r := range filename {
+		if r == ' ' || r == unicode.ReplacementChar || !unicode.IsPrint(r) {
```

Comment:
What is the idea behind also quoting spaces and the unicode replacement characters? Both won't mess up the terminal. That would also make the output change much less invasive.

Reply:
I can drop the quoting of spaces, but the Unicode replacement char is what you get when the input is not UTF-8.

Reply:
Dropped quoting of spaces, i.e., of the ASCII space. Tabs and other whitespace are still quoted.